src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    display: flex;
+    align-items: center;
+    margin-left: 50px;
+    margin-top: 5px;
```

Comment:
Let's use shorthand property margin instead. Also, it would be better to control these spaces with padding in the header element, in this case, you can get rid of the margin-right in navigation.